examples/apple/coreml/scripts/export.py

```diff
@@ -110,9 +123,10 @@ def export_lowered_module_to_executorch_program(lowered_module, example_inputs):
     return exec_prog
 
 
-def save_executorch_program(exec_prog, model_name, compute_unit):
+def save_executorch_program(exec_prog, model_name, compute_unit, quantize):
     buffer = exec_prog.buffer
-    filename = f"{model_name}_coreml_{compute_unit}.pte"
+    data_type = "quantize" if quantize else "fp"
+    filename = f"{model_name}_coreml_{compute_unit}_{data_type}.pte"
```

Comment:
I think this line is responsible for the coreml failure in trunk https://hud.pytorch.org/pytorch/executorch/commit/77e7ad10903ed3bcf16e41d04a5e595ecaa1458b.  It changes the export model file name from `mv3_coreml_all.pte` to `mv3_coreml_all_fp.pte`.  The name is used in many different tests https://github.com/search?q=repo%3Apytorch%2Fexecutorch+mv3_coreml_all&type=code.

cc @SS-JIA @cccclai

Reply:
Trying to revert the change fixes the failures https://github.com/pytorch/executorch/pull/5906

Reply:
ah good finding. Sorry I didn't realize the name was hardcoded. Approve the revert change.